documentation/docusaurus.config.js

```diff
@@ -73,8 +72,8 @@ const config = {
         copyright: `Copyright © ${new Date().getFullYear()} Permit, Inc.`,
       },
       prism: {
-        theme: require("prism-react-renderer/themes/nightOwl"),
-        // darkTheme: require("prism-react-renderer/themes/nightOwlLight"),
+            theme: require('prism-react-renderer').themes.nightOwl,
+            additionalLanguages: ['bash']
```

Comment:
I don't think it's necessary

Reply:
This is part of the changes for the new version of docosourus